File: APIManga/Model/Image.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace APIManga.Model
{
	public class Image
	{
		[Key]
		public int Id { get; set; }
		public int PageId { get; set; }
		public Page Page { get; set; }
		public string ImageUrl { get; set; }
		public int ImageOrder { get; set; }
	}
}

File: backend/app/models/image.py
from sqlalchemy import Column, Integer, String, ForeignKey
from sqlalchemy.orm import relationship
from app.database.connection import Base


class Image(Base):
    __tablename__ = "image"

    id = Column(Integer, primary_key=True)
    page_id = Column(Integer, ForeignKey("page.id"))
    image_url = Column(String)
    image_order = Column(Integer)

    # RELAÇÕES
    page = relationship(
        "Page",
        back_populates="images"
    )